src/libs/ErrorUtils.js

```diff
@@ -35,7 +36,23 @@ function getAuthenticateErrorMessage(response) {
     }
 }
 
+/**
+ * @param {Object} onyxData
+ * @param {Object} onyxData.errors
+ * @returns {String}
+ */
+function getLatestErrorMessage(onyxData) {
```

Comment:
I know we did this before [here](https://github.com/Expensify/App/pull/10556/files#diff-e51fb1efdb0fbb2f3f94e412595749301118191aaf0522067a137917935009abL154), but I don't get why would we only care about one error message

Reply:
👍 yeah I was wondering about this too actually... won't errors also be keyed by microtime so we should be able to sort based on those values - not that we need to necessarily?

Reply:
So I actually took this logic from the LoginForm [here](https://github.com/Expensify/App/blob/41ec9bd371c4f2c2ea63a1ecf922abb6e9789bd5/src/pages/signin/LoginForm.js#L154-L160), which displays the latest error. I simply moved that logic into `getLatestErrorMessage` since I thought it made sense to reuse them here, since they're all part of the same flow.

Doesn't it make sense to only display the latest error though? Why would a user want to know about the other previous errors if they're unrelated to the last action they took?

Reply:
cc @iwiznia @bondydaa ^